src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Details from './components/Display';
+import Heading from './components/Header';
+import Home from './components/Home';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div>
+        <Heading />
+        <div>
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/details" element={<Details />} />
```

Comment:
- [ ] All your category routes have the same `details` as their route, but EACH item is required to have its own unique route. Then you can fetch the item using the unique identifier.

  You should have something like this:
  ```
  path="details/:id"
  ```
  or
  ```
  path=":id"
  ```

  Suggestion: You can consider looking at the `useParams` from the `react-router-dom` package.

  ![idip](https://user-images.githubusercontent.com/81468675/206130859-2a019af7-92db-4bb3-8312-fc8fc44d97cc.jpg)